README.md

```diff
@@ -4,35 +4,151 @@ Service catalog infrastructure stack
 
 This stack will create Amazon vpc, a bastion ec2 server and various components you might require to deploy other stacks in production.
 
-Giving an overview of components available :
+# Architecture
 
-  * AWS S3 bucket to upload backups
-  * AWS S3 bucket to upload deployment artefacts
-  * AWS SSH key pair
-  * AWS EC2 bastion server
-  * AWS VPCs and subnets
-  * AWS SES service to allow your EC2 servers send system mails
-  * AWS IAM roles and admin policies
+<p align="center">
+<img src="https://raw.githubusercontent.com/cycloid-community-catalog/stack-infrastructure/master/diagram.jpeg" width="400">
```

Comment:
we can change this to:

`<img src="diagram.jpeg" width="400">`

or does it make sense if we create a `_docs` folder with all this kind of stuff inside?

Reply:
nice catch for the url, I will change it everywere.

for the docs directory why not let's move all images in it

Reply:
100% agree